news-quiz.php

```diff
@@ -37,60 +37,67 @@ function news_quiz_shortcode( $atts ){
 	$source = $atts['source'];
 
 	$answerstyle = $atts['answerstyle'] ? $atts['answerstyle'] : 'bullet';
-	if( !in_array( $answerstyle, array('alpha','bullets','roman','numbers') ) ) 
+	if ( ! in_array( $answerstyle, array( 'alpha', 'bullets', 'roman', 'numbers' ), true ) ) {
 		$answerstyle = 'bullet';
+	}
 
 	$layout = $atts['layout'] ? $atts['layout'] : 'fullwidth';
-	if( !in_array( $layout, array('fullwidth','sidebar') ) ) 
+	if ( ! in_array( $layout, array( 'fullwidth', 'sidebar' ), true ) ) {
 		$layout = 'fullwidth';
+	}
 
 	$align = $atts['align'] ? $atts['align'] : 'alignnone';
-	if( $layout == 'fullwidth' || !in_array( $align, array('alignnone','aligncenter','alignright','alignleft') ) )
+	if ( 'fullwidth' === $layout || ! in_array( $align, array( 'alignnone', 'aligncenter', 'alignright', 'alignleft' ), true ) ) {
 		$align = 'alignnone';
+	}
 
 	/* 3: Return container */
-	
-	$ret  = "";
+
+	$ret  = '';
 
 	// .news-quiz
 	$ret .= "<aside class='news-quiz news-quiz-quiz news-quiz-$align news-quiz-$layout'>";
 
 	// .news-quiz-header
-	if($description || $title) {
+	if ( $description || $title ) {
 
 		$ret .= "<header class='news-quiz-header'>";
-	
-		if( $title )
-			$ret .= "<h3 class='news-quiz-headline'>" . $title . "</h3>";
-		if( $description )
-			$ret .= "<p class='news-quiz-lede'>" . $description . "</p>";
-	
-		$ret .= "</header>";
+
+		if ( $title ) {
+			$ret .= "<h3 class='news-quiz-headline'>" . $title . '</h3>';
+		}
+		if ( $description ) {
+			$ret .= "<p class='news-quiz-lede'>" . $description . '</p>';
+		}
+
+		$ret .= '</header>';
 
 	}
 
 	// .news-quiz-content
-	$ret .= "<div ";
+	$ret .= '<div ';
 		$ret .= "class='news-quiz-quizbox news-quiz-content news-quiz-quizbox-list-$answerstyle' ";
 		$ret .= "id='quizcontainer' ";
 		$ret .= "data-key='$key' ";
-	$ret .= "></div>";
+	$ret .= '></div>';
 
 	// .news-quiz-footer
-	if( $byline || $source ) {
-		
+	if ( $byline || $source ) {
+
 		$ret .= "<div class='news-quiz-footer'>";
-	
-		if( $byline )
-			$ret .= "<span class='news-quiz-byline'>" . $byline . "</span>";
-		if( $source ) 
-			$ret .= "<span class='news-quiz-source'>Source: " . $source . "</span>";
 
-		$ret .= "</div>";
+		if ( $byline ) {
+			$ret .= "<span class='news-quiz-byline'>" . $byline . '</span>';
```

Comment:
Why does the second string here switch from double to single quotes, but not the first?